__tests__/integration.js

```diff
@@ -51,6 +59,7 @@ addTest('https://git@github.com/stevemao/left-pad.git'); // git url, with userna
 addTest('https://github.com/yarnpkg/yarn/releases/download/v0.18.1/yarn-v0.18.1.tar.gz'); // tarball
 addTest('https://github.com/bestander/chrome-app-livereload.git'); // no package.json
 addTest('bestander/chrome-app-livereload'); // no package.json, github, tarball
+addTest('react-scripts', {strict: true}); // many peer dependencies, there shouldn't be any peerDep warnings
```

Comment:
I wonder if we should fix this version in case the react-scripts package does get updated?

Reply:
Ahh, another good insight. Update coming.